### DISCUSSION TITLE
A telegram bot that sends customized job opening notifications.

### DISCUSSION THREAD

**Discussion by tushar-amrit-6** [scaleracademy/open-source-projects#30] [Ideas]
## PROJECT SUMMARY
A notification bot especially for fresh graduates and summer internship seekers that prevents our LinkedIn feed from #interested spamming. User can customize the type of job opening they are looking for according to their qualification and pass-out year. Also user can set the companies they are targeting(default settings being FAANG 😉).

## MVP
-Alerts for new openings.
-Easy tracking of job profiles applied for.
-Reminders if the job is not applied and deadline is near.
-Filtered links to various sites like Leetcode Discussions, Glassdoor for details related to the profile and company.

## Add Ons
-Working Professionals can give their contact details through the bot to find candidates for referrals.

**Comment by jyotirmaya907** [comment]
Amazing idea!!
